templates/profile/prometheus/exporter/node/rubyHealthCheck-cronjob.sh.erb

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
We usually put the small "managed by puppet" banner on files we drop in.

Reply:
added a comment.